modtools_frozen.yml

```diff
@@ -0,0 +1,60 @@
+- path: github.com/DataDog/datadog-go
+  minVersion: v0.0.0-20180330214955-e67964b4021a
+  validUntil: 2029-08-04T16:27:45+03:00
+- path: github.com/Shopify/sarama
+  minVersion: v1.16.0
+  validUntil: 2029-08-04T16:27:55+03:00
+- path: github.com/eapache/go-resiliency
+  minVersion: v1.1.0
+  validUntil: 2029-08-04T16:28:36+03:00
+- path: github.com/eapache/go-xerial-snappy
+  minVersion: v0.0.0-20160609142408-bb955e01b934
+  validUntil: 2029-08-04T16:28:44+03:00
+- path: github.com/rcrowley/go-metrics
+  minVersion: v0.0.0-20180503174638-e2704e165165
+  validUntil: 2029-08-04T16:29:04+03:00
+- path: github.com/spf13/afero
+  minVersion: v1.1.2
+  validUntil: 2029-08-04T16:29:18+03:00
+- path: github.com/influxdata/influxdb1-client
+  minVersion: v0.0.0-20190402204710-8ff2fc3824fc
+  validUntil: 2029-08-04T16:29:55+03:00
+- path: github.com/kubernetes/helm
+  minVersion: v2.9.0+incompatible
+  validUntil: 2029-08-04T16:30:19+03:00
+- path: github.com/pierrec/xxHash
+  minVersion: v0.1.1
+  validUntil: 2021-05-25T16:30:26+03:00
+- path: github.com/pierrec/lz4
+  minVersion: v1.0.2-0.20171218195038-2fcda4cb7018
+  validUntil: 2021-05-25T16:30:45+03:00
+- path: github.com/andybalholm/cascadia
+  minVersion: v1.1.0
+  validUntil: 2029-08-04T16:32:04+03:00
+- path: gopkg.in/guregu/null.v3
+  minVersion: v3.3.0
+  validUntil: 2029-08-04T16:32:44+03:00
+- path: gopkg.in/yaml.v3
+  minVersion: v3.0.0-20200313102051-9f266ea9e77c
+  validUntil: 2029-08-04T16:32:56+03:00
+- path: github.com/manyminds/api2go
+  minVersion: v0.0.0-20180125085803-95be7bd0455e
+  validUntil: 2029-08-04T16:33:07+03:00
+- path: github.com/mitchellh/mapstructure
+  minVersion: v1.1.2
+  validUntil: 2029-08-04T16:51:38+03:00
+- path: golang.org/x/sys
+  minVersion: v0.0.0-20201204225414-ed752295db88
+  validUntil: 2029-08-04T17:00:51+03:00
+- path: google.golang.org/genproto
+  minVersion: v0.0.0-20200903010400-9bfcb5116336
+  validUntil: 2029-08-04T17:01:00+03:00
+- path: google.golang.org/protobuf
+  minVersion: v1.25.1-0.20200805231151-a709e31e5d12
+  validUntil: 2029-08-04T17:01:13+03:00
+- path: github.com/golang/protobuf
+  minVersion: v1.4.2
```

Comment:
Why are these frozen?

Reply:
They are dependency of the `grpc` and I think we should update them when `grpc` updates them instead of whenever they get an update